handshaken error during authentication

Hello

Here is a crash I get on a standard installation with socket.io-client@1.2.1

```
[2014-12-01 22:49:12.496] [ERROR] console - TypeError: Cannot read property 'handshaken' of undefined
    at Object.exports.handleMessage [as hook_fn] (/var/www/etherpad-lite/node_modules/ep_ldapauth/ep_ldapauth.js:176:43)
    at normalizedhook (/var/www/etherpad-lite/src/static/js/pluginfw/hooks.js:15:27)
    at hookCallWrapper (/var/www/etherpad-lite/src/static/js/pluginfw/hooks.js:21:14)
    at /var/www/etherpad-lite/src/static/js/pluginfw/hooks.js:88:7
    at /var/www/etherpad-lite/src/node_modules/async/lib/async.js:249:17
    at /var/www/etherpad-lite/src/node_modules/async/lib/async.js:125:13
    at Array.forEach (native)
    at _each (/var/www/etherpad-lite/src/node_modules/async/lib/async.js:46:24)
    at async.each (/var/www/etherpad-lite/src/node_modules/async/lib/async.js:124:9)
    at _asyncMap (/var/www/etherpad-lite/src/node_modules/async/lib/async.js:248:13)
```

Any idea to solve it?

--- a/ep_ldapauth.js
+++ b/ep_ldapauth.js
@@ -172,9 +172,8 @@ exports.handleMessage = function(hook_name, context, cb) {
     if (!context.message.token) {
       console.debug('ep_ldapauth.handleMessage: intercepted CLIENT_READY message has no token!');
     } else {
-      var client_id = context.client.id;
-      if ('user' in context.client.manager.handshaken[client_id].session) {
-        var displayName = context.client.manager.handshaken[client_id].session.user.displayName;
+      if ('user' in context.client.client.request.session) {
+        var displayName = context.client.client.request.session.user.displayName;
         if(settings.users.ldapauth.anonymousReadonly && !displayName) displayName = 'guest';
         console.debug('ep_ldapauth.handleMessage: intercepted CLIENT_READY message for client_id = %s, setting username for token %s to %s', client_id, context.message.token, displayName);
         ldapauthSetUsername(context.message.token, displayName);

